tests/reporters/test_autopay_multicalls.py

```diff
@@ -29,10 +29,6 @@ async def test_get_current_tips(setup_autopay_call):
     tips = await calls.get_current_tip(require_success=False)
     assert tips["eth-usd-spot"] is None
 
-    with pytest.raises(ContractLogicError):
```

Comment:
Why was this taken out?

Reply:
before this function raised this error, instead now I catch it